data/actionsmap/yunohost.yml

```diff
@@ -1584,7 +1599,7 @@ tools:
                 - PUT /domains/main
             arguments:
                 -n:
-                    full: --new-domain
+                    full: --new-main-domain
```

Comment:
Probably need to propagate this on the controller in the webadmin : https://github.com/YunoHost/yunohost-admin/blob/stretch-unstable/src/js/yunohost/controllers/domains.js#L309

Reply:
Ah yeah, I have code alredy done for that, forgot to push it :sweat:

Reply:
https://github.com/YunoHost/yunohost-admin/pull/260 done